src/backend/distributed/shared_library_init.c

```diff
@@ -626,6 +626,18 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_unique_job_ids",
+		gettext_noop("Enables unique job IDs by prepending the local process ID and "
+					 "group ID. This should usually be enabled, but can be disabled "
+					 "for repeatable output in regression tests."),
+		NULL,
+		&EnableUniqueJobIds,
+		true,
+		PGC_USERSET,
+		GUC_NO_SHOW_ALL,
```

Comment:
Probably not necessary but, adding GUC_SUPERUSER_ONLY may make sense, up to you.